test/ownership/Superuser.test.js

```diff
@@ -44,6 +45,16 @@ contract('Superuser', function (accounts) {
       const currentOwner = await this.superuser.owner();
       currentOwner.should.be.equal(newOwner);
     });
+
+    it('should change owner after the owner transfers the ownership', async function () {
+      await expectEvent.inTransaction(
+        this.superuser.transferOwnership(finalOwner, { from: newOwner }),
```

Comment:
I was confused that `newOwner` was sending this tx, but I saw that you're using the same `Superuser` instance for all tests! You should create the `Superuser` instance in a `beforeEach` block instead of `before`, so as to have the tests be independent of one another.

Then it will be `firstOwner` sending this tx.